/**:
  ros__parameters:
    camera_name: camera
    gain: 10.0
    gain_lower_limit: 1.0
    gain_upper_limit: 10.0
    exposure: 6000.0
    gamma: 0.5
    camera_info_package: arena_camera_ros2
    camera_info_file: camera_intrinsics/test_camera_info.yaml
